WcaOnRails/app/views/admin/compute_auxiliary_data.html.erb

```diff
@@ -0,0 +1,19 @@
+<% provide(:title, 'Compute Auxiliary Data') %>
+
+<div class="container">
+  <%= render layout: "nav" do %>
+    <h2><%= yield(:title) %></h2>
+    <% if @auxiliary_data_start.date.nil? %>
+      <%= alert :danger, "Oh dear! The data has never been computed!" %>
+    <% elsif @auxiliary_data_end.date.nil? %>
+      <%= alert :info, "The data is being computed. Thanks for checking =)" %>
+    <% else %>
+      <%= alert :success, "Data was last computed #{time_ago_in_words @auxiliary_data_end.date} ago and took #{distance_of_time_in_words @auxiliary_data_start.date, @auxiliary_data_end.date}" %>
+    <% end %>
+    <% unless @auxiliary_data_start.date.present? && @auxiliary_data_end.date.nil? # Show unless the data is being computed. %>
```

Comment:
This took me a while to understand. I think this would be a good method to put on `TimedApplicationJob` so we could instead write this as `ComputeAuxiliaryData.in_progress?`